src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/validsendtointerloc/ValidSendToInterlocMidEventHandlerTest.java

```diff
@@ -64,9 +65,13 @@ public void setUp() {
         openMocks(this);
         handler = new ValidSendToInterlocMidEventHandler(generateFile, templateId);
 
+        Hearing hearing = getHearing(1);
```

Comment:
Same here. If you've changed to the postponement request should there not be a test reflecting this?

Reply:
I see that there's a postponementRequest attached. Could you please make sure that there's a test ensuring the appropriate additionalRequestDetails have been attached?

Reply:
Test has been added.